src/nifs.rs

```diff
@@ -213,43 +311,265 @@ mod tests {
     W2: &R1CSWitness<E>,
   ) {
     // produce a default running instance
-    let mut r_W = RelaxedR1CSWitness::default(shape);
-    let mut r_U = RelaxedR1CSInstance::default(ck, shape);
+    let mut running_W = RelaxedR1CSWitness::default(shape);
```

Comment:
Nit: Any reason to rename these?

Reply:
`r_W` is used other places to denote the blind for the witness vector. It was just a clarity thing. If you think its better the other way, I can change it back.

Reply:
Good point! Let's keep the change.